.github/workflows/breaking-change-check.yml

```diff
@@ -72,6 +72,9 @@ jobs:
       - name: Install beta dependencies
         if: ${{ matrix.flavor == 'beta' && github.event_name == 'pull_request' && !contains(github.event.pull_request.labels.*.name, 'breaking-change') }}
         run: rush switch-flavor:beta
+      - name: Install beta-release dependencies
```

Comment:
Needed to cherrypick this fix to github action: https://github.com/Azure/communication-ui-library/pull/3867